public/app/features/alerting/unified/components/AlertingPageWrapper.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { FC } from 'react';
+import Page from 'app/core/components/Page/Page';
+import { getNavModel } from 'app/core/selectors/navModel';
+import { useSelector } from 'react-redux';
+import { StoreState } from 'app/types/store';
+import { useStyles } from '@grafana/ui';
+import { css } from '@emotion/css';
+import { GrafanaTheme } from '@grafana/data';
+
+interface Props {
+  pageId: string;
+  isLoading?: boolean;
+}
+
+export const AlertingPageWrapper: FC<Props> = ({ children, pageId, isLoading }) => {
+  const navModel = getNavModel(
+    useSelector((state: StoreState) => state.navIndex),
+    pageId
+  );
+
+  const styles = useStyles(getStyles);
+
+  return (
+    <Page navModel={navModel} className={styles.page}>
+      <Page.Contents isLoading={isLoading}>{children}</Page.Contents>
+    </Page>
+  );
+};
+
+export const getStyles = (theme: GrafanaTheme) => ({
+  page: css`
+    .page-container {
+      max-width: ${theme.breakpoints.xxl};
+    }
+  `,
```

Comment:
instead of customizing and overriding the Page styles from outside, think this should be a property on Page component that controls the maxWidth. Is there a reason for needing more width? these views need to work on laptop / smaller screens as well. 

In the coming v8 design updates we are going to change this a bit so the nav tabs are vertically stacked on the left side. Might go with a full width design for the page content box, not sure yet
![Screenshot from 2021-04-06 14-45-49](https://user-images.githubusercontent.com/10999/113712777-d2e80e80-96e6-11eb-9ad5-b95ff8fbbccb.png)

Reply:
> instead of customizing and overriding the Page styles from outside, think this should be a property on Page component that controls the maxWidth

Done!

> Is there a reason for needing more width? these views need to work on laptop / smaller screens as well.

It's designed to take advantage of wider area. Will look good enough laptops & smaller, but in current state not usable on mobile. I'll bring this up with UX, but not sure that we'll have dev bandwidth to adapt for mobile for 8...

> In the coming v8 design updates we are going to change this a bit so the nav tabs are vertically stacked on the left side. Might go with a full width design for the page content box, not sure yet

Full width sounds great to me, it's a shame to waste that screen space :)

Reply:
> but in current state not usable on mobile. I

The alert lists / alert state (instances firing) seem like important lists to work well on mobile. But the edit rule does not need to work on mobile